.github/workflows/express-build.yml

```
@@ -0,0 +1,110 @@
+# GitHub Action Workflow for building on Linux and Windows
+
+name: ExpressJS Express Build
```

```suggestion
name: Test
```

This whole thing is just the express tests, lets just rename the file and the title.

---

I had not plan with the naming.  Let's change it.

---

I've made this change (changing the file name)